app/controllers/organizations/staff/people_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Organizations::Staff::PeopleController < Organizations::BaseController
+  skip_before_action :verify_and_set_current_person
```

Comment:
Update this callback if you do implement this https://github.com/rubyforgood/homeward-tails/pull/1354/files#r2023859335